### PR TITLE
RUM-7484 Add placeholder for unknown SwiftUI content

### PR DESCRIPTION
**Pull request by maxep**
### How?

Return placeholder wireframe when an unknown SwiftUI content is encountered.

### Review checklist
- [x] Feature or bugfix MUST have appropriate tests (unit, integration)
- [x] Make sure each commit and the PR mention the Issue number or JIRA reference
- [ ] Add CHANGELOG entry for user facing changes
- [ ] Add Objective-C interface for public APIs (see our [guidelines](https://datadoghq.atlassian.net/wiki/spaces/RUMP/pages/3157787243/RFC+-+Modular+Objective-C+Interface#Recommended-solution) [internal]) and run `make api-surface`)

### CODE REVIEW

**Review by mariedm — DISMISSED** [review]

**Review by mariedm — APPROVED** [review]

File: DatadogSessionReplay/Tests/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/NodeRecorders/SwiftUI/SwiftUIWireframesBuilderTests.swift [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import XCTest
+import SwiftUI
+import DatadogInternal
+import TestUtilities
+
+@_spi(Internal)
+@testable import DatadogSessionReplay
+
+@available(iOS 13.0, tvOS 13.0, *)
+class SwiftUIWireframesBuilderTests: XCTestCase {
```
Comment (msrutek-paylocity) [inline DatadogSessionReplay/Tests/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/NodeRecorders/SwiftUI/SwiftUIWireframesBuilderTests.swift:19]: `final`

**Pull request merged**